ivy/container/general.py

```diff
@@ -50,3 +51,106 @@ def clip_vector_norm(
             ),
             out,
         )
+
+    def all_equal(
+        self,
+        x,
+        equality_matrix: bool = False
+    ):
+        """Determines whether the inputs are all equal.
+
+        Parameters
+        ----------
+        xs
+            inputs to compare.
+        equality_matrix
+            Whether to return a matrix of equalities comparing each
+            input with every other.
+            Default is False.
+
+        Returns
+        -------
+        ret
+            Boolean, whether or not the inputs are equal, or matrix array of booleans
+            if equality_matrix=True is set.
+
+        Instance Method Examples
```

Comment:
Both examples seem to be doing the same thing, please remove one example from each so that we have one example for `With one ivy.Container instances` and one example for `With multiple ivy.Container instances`

Reply:
Also, the header can simply be `Examples`